lighthouse-core/gather/driver.js

```diff
@@ -596,6 +619,26 @@ class Driver {
   }
 
   /**
+   * @return {!Promise<number>}
+   */
+  _getOrCreateIsolatedContextId() {
```

Comment:
`_acquireIsolatedContextId()`  ?

Reply:
discussed offline, Paul's a huge fan of `getOrCreate` now 🚗  🎏  🎉

Reply:
lol